webapp/sources/rudder/rudder-web/src/main/javascript/rudder/rudder-datatable.js

```diff
@@ -1192,7 +1193,9 @@ var allColumns = {
                  }
   , "Score details" :
     function(scoreId) {
-      var title = "Score '"+scoreId+"'"
+      var score = scoreList.find((element) => element.id === scoreId);
+      var scoreName = score !== undefined ? score.name : scoreId;
+      var title = "'"+scoreName+"' Score";
```

Comment:
I think since we have a "name" it's more natural to read it when have :
```suggestion
      var title = scoreName + " Score";
```
![image](https://github.com/Normation/rudder/assets/65616064/8bf239a7-5e0b-4269-a0b7-66874c817dce)